includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -498,6 +498,57 @@ public static function orders( $data = null ) {
 			$total   = $meta['_edd_payment_total'][0];
 		}
 
+		// Build up the order address data. Actual insertion happens later, but we need this now to figure out the tax rate.
+		$order_address_data = array(
+			'name'         => trim( $user_info['first_name'] . ' ' . $user_info['last_name'] ),
+			'address'      => isset( $user_info['address']['line1'] )   ? $user_info['address']['line1']   : '',
+			'address2'     => isset( $user_info['address']['line2'] )   ? $user_info['address']['line2']   : '',
+			'city'         => isset( $user_info['address']['city'] )    ? $user_info['address']['city']    : '',
+			'region'       => isset( $user_info['address']['state'] )   ? $user_info['address']['state']   : '',
+			'country'      => isset( $user_info['address']['country'] ) && array_key_exists( strtoupper( $user_info['address']['country'] ), edd_get_country_list() )
+				? $user_info['address']['country']
+				: '',
+			'postal_code'  => isset( $user_info['address']['zip'] )     ? $user_info['address']['zip']     : '',
+			'date_created' => $date_created_gmt,
+		);
+
+		// Remove empty data.
+		$order_address_data = array_filter( $order_address_data );
+
+		$tax_rate_id = null;
+		$tax_rate = isset( $meta['_edd_payment_tax_rate'][0] )
+			? (float) $meta['_edd_payment_tax_rate'][0]
+			: 0.00;
+
+		/*
+		 * Previously tax rates were stored as a decimal (e.g. `0.2`) but they're now stored as a percentage
+		 * (e.g. `20`). So we need to convert.
+		 */
+		if ( $tax_rate < 1 ) {
+			$tax_rate = $tax_rate * 100;
+		}
+
+		$set_tax_rate_meta = false;
+
+		if ( ! empty( $tax_rate ) && ! empty( $order_address_data['country'] ) ) {
```

Comment:
I need to spend more time here digging into why, but this does not seem to be evaluating as true for me, even though it should. Any error logging I'm doing inside this function is not happening, even if the country is set and the tax rate is not empty. I'll dig more tomorrow.